src/transforms/babel.js

```diff
@@ -26,8 +26,8 @@ const JSX_PRAGMA = {
   hyperapp: 'h'
 };
 
-async function babelTransform(asset) {
-  let config = await getConfig(asset);
+async function babelTransform(asset, isProdBuild) {
```

Comment:
shouldn't be necessary as asset.options.production should return the same as `isProdBuild `

Reply:
Refactored 👌